pkg/acceptance/util.go

```diff
@@ -569,7 +569,18 @@ func testDocker(
 		containerConfig.Env = append(containerConfig.Env, "PGHOST="+l.Hostname(0))
 	}
 	hostConfig := container.HostConfig{NetworkMode: "host"}
-	return l.OneShot(ctx, postgresTestImage, types.ImagePullOptions{}, containerConfig, hostConfig, "docker-"+name)
+	if err := l.OneShot(
+		ctx, postgresTestImage, types.ImagePullOptions{}, containerConfig, hostConfig, "docker-"+name,
+	); err != nil {
+		return err
+	}
+	// Clean up the log files if the run was successful.
+	if l.LogDir != "" {
```

Comment:
Hmm, this would be pretty bad if we shared `l.LogDir` between the tests, which I don't think we do, but running a test where `l.LogDir = $HOME` would be pretty bad. Maybe this is too risky given that it doesn't buy us much?